lib/helpers.js

```diff
@@ -88,15 +87,25 @@ exports.releaseDownloadUrl = function (name, version) {
     return url.format({
         protocol: 'https:',
         hostname: 'github.com',
-        pathname: path.join(
```

Comment:
Because `path.join` will use `\` or `/` depending on the platform, but URLs must use `/` all the time.